src/diff/props.js

```diff
@@ -14,7 +14,7 @@ export function diffProps(dom, newProps, oldProps, isSvg, hydrate) {
 	let i;
 
 	for (i in oldProps) {
-		if (!(i in newProps)) {
+		if (typeof newProps[i] === 'undefined') {
```

Comment:
FIY: I checked if `newProps[i] === undefined` would be smaller, but it leads to the same size :tada:

Reply:
Uh, I think we've used this somewhere else, and I hoped to get better size doing the `typeof`. But yeah seems to be the same 😄 

Thanks for looking into it!